docs/registry/index.rst

```diff
@@ -212,7 +212,7 @@ A special sort of access mode is ``web``, which represents some facility related
 to the resource that works in a web browser.  You can ask for a
 “service” for it, too; you will then receive an object that has a
 ``search`` method, and when you call it, a browser window should open
-with the query facility (this uses python's webbrowser module):
+with the query facility (this uses python's web browser module):
```

Comment:
Oh, the module really is called webbrowser, and hence you shouldn't separate it.  If you want to add some verbatim markup, that might prevent future confusion.

Reply:
aaah sorry. Reverted